komodo-core/src/main/resources/org/komodo/repository/initialContent.xml

```diff
@@ -6,6 +6,7 @@
 		<tko:library jcr:primaryType="tko:library" />
 		<tko:environment jcr:primaryType="tko:environment">
             <tko:validation jcr:primaryType="nt:unstructured" />
+            <tko:servers jcr:primaryType="nt:unstructured" />
```

Comment:
I'm wondering if creating the `DefaultServer` node under `tko:servers` would also be a good idea to do here. Wdyt?